constants/fokabotCommands.py

```diff
@@ -118,12 +118,22 @@ async def roll(fro: str, chan: str, message: list[str]) -> str:
 
 
 @command(trigger="!alert", privs=privileges.ADMIN_SEND_ALERTS, hidden=True)
```

Comment:
```suggestion
@command(trigger="!alertall", privs=privileges.ADMIN_SEND_ALERTS, hidden=True)
```

Reply:
super ugly but explicit - no one will make the mistake over !alert instead of !alertu now